### PR TITLE
Weekly Publishing 27.04.2016 - Online

### PR DESCRIPTION
**Pull request by vikram-redhat**

### CODE REVIEW

File: architecture/infrastructure_components/web_console.adoc [hunk 1]
```diff
@@ -49,6 +51,23 @@ link:../../install_config/master_node_configuration.html#master-configuration-fi
...
+link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[*About*
+page on the web console] for use on Linux, MacOSX, and Windows clients
+ifdef::openshift-enterprise,openshift-origin[]
+if your cluster administrator has enabled it
+endif::[]
+:
+
+image::about_page.png["About Page"]
+
+>>>>>>> a2ab3c6... update for atomic registry use case
```
Comment (adellape) [inline architecture/infrastructure_components/web_console.adoc]: @vikram-redhat You've got merge conflict leftovers here.

File: architecture/revhistory_architecture.adoc [hunk 2]
```diff
@@ -4,6 +4,31 @@
...
+
+|Affected Topic |Description of Change
+//Wed Apr 27 2016
+|link:../architecture/core_concepts/routes.html[Core Concepts -> Routes]
+|Corrected footnote placement in the link:../architecture/core_concepts/routes.html#secured-routes[Secured Routes] section.
+
+n|link:../architecture/core_concepts/builds_and_image_streams.html[Core Concepts -> Builds and Image Streams]
+|Added information about maximum name lengths for objects.
+
+n|link:../architecture/core_concepts/projects_and_users.html[Core Concepts -> Projects and Users]
```
Comment (adellape) [inline architecture/revhistory_architecture.adoc]: Some stray 'n's on lines 18 and 21?

Comment (adellape) [inline architecture/revhistory_architecture.adoc]: FYI @tnguyen-rh, in case related to recent script changes.

File: welcome/revhistory_full.adoc [hunk 3]
```diff
@@ -4,6 +4,9 @@
 :data-uri:
 :icons:
 :experimental:
+== Wed Apr 27 2016
+.Architecture
+include::architecture/revhistory_architecture.adoc[tag=architecture_wed_apr_27_2016]
```
Comment (adellape) [inline welcome/revhistory_full.adoc]: This section is above the intro sentence. Needs to get put back in place.

File: architecture/revhistory_architecture.adoc (hunk 2 above)
Comment (tnguyen-rh) [inline architecture/revhistory_architecture.adoc]: @adellape Thanks.  Yes, this is related to the recent changes.  I have submitted a merge request (no.9) w/ a fix for the spurious 'n', and one (no.10) for the "out of place" intro sentence.

**Comment by adellape** [comment]
:+1:

**Pull request merged**